src/Search/Gearman/Command/ImportCommand.php

```diff
@@ -65,6 +68,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return;
         }
 
+        if ($input->getOption('dateFrom') !== null) {
+            $this->dateFrom = new \DateTimeImmutable($input->getOption('dateFrom'));
+            if ($this->dateFrom < (new \DateTimeImmutable('-30 days'))) {
+                throw new \Exception('Date must not be bigger than 30 days');
```

Comment:
Add use statements for `DateTimeImmutable` and `Exception`. Also, elsewhere we use `DateTimeImmutable::createFromFormat(DATE_ATOM, $input->getOption('dateFrom'))`.

See: https://github.com/elifesciences/search/blob/develop/src/Search/Workflow/ResearchArticleWorkflow.php#L117